models/dao/dao_schema.yml

```diff
@@ -0,0 +1,95 @@
+version: 2
+
+models:
+  - name: dao_votes
+    meta:
+      blockchain: ethereum
+      sector: dao
+      contributors: soispoke
+    config:
+      tags: ['ethereum', 'votes', 'dao', 'cross-chain']
+    description: >
+      DAO votes on all chains across all contracts and versions
+    columns:
+      - &blockchain
+        name: blockchain
+        description: "Blockchain which the DAO is deployed"
+      - &project
+        name: project
+        description: "Project name of the DAO"
+      - &version
```

Comment:
How do protocol versions map here? Does e.g. uniswap have different DAO contracts for each version of the protocol?

Reply:
I'm confident the active, tally contract is specifically related to uniswap v3, and DAO contracts existed for previous versions, see screenshot of governoralpha contract for uniswap_v2

I think those should be added as well, but prob in a different PR
<img width="388" alt="Capture d’écran 2022-11-14 à 08 40 55" src="https://user-images.githubusercontent.com/66172107/201602489-7101b5d4-bfa8-4c66-8b28-f2a4423ed2ee.png">

Reply:
Sounds good, yeah I think it actually might depend on what projects choose since this contract can be modified to suit particular need.